### PR TITLE
[W-8857319] - Fixing hardcoded string reference that prevent label audit to pass

### PR DESCRIPTION
**Pull request by jdavilasf**
This PR remove the hardcoded reference made on the Disbursements aura component. The Default Model now is being populated by a new helper method. 

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
- [x] ~~Any net new LWC work has JEST test coverage 50% or above~~
- [x] ~~Default Sa11y tests pass for all LWC components~~
- [x] 🔒 ~~Secure both Front-end (LWC) & back-end (Apex) as necessary~~
- [x] 🔑 ~~Grant users access in Permission Sets (Object, Field, Apex Class) as necessary~~
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
- [x] Make sure that ACs are updated (if any gaps)
- [x] **All acceptance criteria have been met**
    - [x] Developer
    - [ ] Code Reviewer
- [x] ~~Pull Request contains draft release notes~~
- [x] ~~Labels, help text, and customer facing messages are reviewed by Docs~~
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by jdavilasf** [comment]
[W-8857319](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000095wvHIAQ/view)

**Comment by spelak-salesforce** [comment]
@sella24 , if this change looks good to pass the string label audit, can you merge this into the parent branch?

cc: @jdavilasf

File: force-app/main/default/aura/disbursementsCreate/disbursementsCreateHelper.js [hunk 1]
```diff
@@ -1,62 +1,12 @@
...
-        disbursements: [],
-        uiMessages: []
-    },
-
+    DEFAULT_MODEL: {},
     VIEW_MODEL: {},
 
     init: function (cmp) {
+        // Set Default Model Values
+        this.setDefaultModel(cmp);
```
Comment (sella24) [inline force-app/main/default/aura/disbursementsCreate/disbursementsCreateHelper.js:7]: @jdavilasf : 
Can setDefaultModel be moved to the helper and called here without needed to set the component's attribute ?
Something like :
`this.DEFAULT_MODEL = JSON.parse(JSON.stringify(helper.setDefaultModel(cmp)));`

Comment (jdavilasf) [reply force-app/main/default/aura/disbursementsCreate/disbursementsCreateHelper.js:7]: I think we can set the `this.DEFAULT_MODEL` up directly on the `setDefaultModel` method. If that's possible we won't need the defaultModel attribute. Let me try it. Good suggestion @sella24 👍

Comment (jdavilasf) [reply force-app/main/default/aura/disbursementsCreate/disbursementsCreateHelper.js]: All set @sella24 . I updated the helper and removed the aura attribute `defaultModel`. Now we populate the `DEFAULT_MODEL` object directly from the method. I tested it again and works well. Feel free to check again. Thanks!

**Review by sella24 — APPROVED** [review]
Great job Julio. Looks good.

**Pull request merged**